packages/integration-karma/test/light-dom/synthetic-shadow/index.spec.js

```diff
@@ -39,9 +39,12 @@ describe('Light DOM + Synthetic Shadow DOM', () => {
             expect(nodes.p.parentElement).toEqual(nodes.consumer);
             expect(nodes.consumer.parentElement).toEqual(elm);
         });
-        // TODO [#2424]: Fails because the <p> doesn't have an owner key.
-        xit('getRootNode', () => {
-            expect(nodes.p.getRootNode()).toEqual(document);
+        // Issue [#2424]: Fails because the <p> doesn't have an owner key.
+        // Synthetic shadow is no longer being changed. This test now verifies
+        // that the existing behavior does not regress.
+        it('getRootNode', () => {
+            // expect(nodes.p.getRootNode()).toEqual(document); // correct behavior
```

Comment:
I left this in so there's a record of the desired behavior in case our understanding of this issue changes in the near future.